### DISCUSSION TITLE
Google Cloud Docker

### DISCUSSION THREAD

**Discussion by Thinline-Dynamic-Solutions** [chuot/rdio-scanner#336] [Q&A]
I am trying to move my rdio-scanner server over to Google Cloud VM with container optimized OS. which supports docker. I keep running into the issue /app/data/ no writer permissions. I'm very new to the docker platform and was wondering if anyone has been successful in setting up through SSH or has any insights to accomplish this.

**Comment by tadscottsmith** [comment]
What command are you using to start the docker container? And what do the permissions look like on the /app/data directory?

**Comment by Thinline-Dynamic-Solutions** [comment]
I may play around with this again this weekend. I aborted mission and don't
remember a whole lot of the headache experience. I started using Azure as a
backup and may stick to it.
